payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncherViewModel.kt

```diff
@@ -141,7 +141,7 @@ internal class PaymentLauncherViewModel @Inject constructor(
                             )
                         }
                     } else {
-                        authenticatorRegistry.getAuthenticator(intent).authenticate(
+                        authenticatorRegistry.getAuthenticator(intent).nextAction(
```

Comment:
Since the previous function name was a verb (authenticate), I would expect this to still be a verb -- e.g. something like `performNextAction`. Can you rename this?